web/src/main/resources/application.properties-EXAMPLE

```diff
@@ -17,3 +17,14 @@ slack.url=
 
 # required OncoTree version for startup/recache
 required.oncotree.version=oncotree_latest_stable
+
```

Comment:
this is duplicate file. Please remove the one from the core module and keep this one only.

Reply:
I think the duplicate you are referencing in core is for unit tests (not all properties are needed) and one is for the actual web application (all properties required).